lib/Runtime/Library/JavascriptLibraryBase.h

```diff
@@ -298,6 +299,21 @@ namespace Js
         Field(JavascriptSymbol*) symbolToStringTag;
         Field(JavascriptSymbol*) symbolUnscopables;
 
+    public:
+        typedef void (CALLBACK *HostPromiseRejectionTrackerCallback)(Var promise, Var reason, bool handled, void *callbackState);
+
+        void SetNativeHostPromiseRejectionTrackerCallback(HostPromiseRejectionTrackerCallback function, void *state)
+        {
+            this->nativeHostPromiseRejectionTracker = function;
+            this->nativeHostPromiseRejectionTrackerState = state;
+        }
+
+        void CallNativeHostPromiseRejectionTracker(Var promise, Var reason, bool handled);
+
+    private:
+        FieldNoBarrier(HostPromiseRejectionTrackerCallback) nativeHostPromiseRejectionTracker = nullptr;
+        Field(void *) nativeHostPromiseRejectionTrackerState; // TODO (doilij) = nullptr; // would this cause problems?
```

Comment:
I don't understand the TODO

Reply:
I just noticed this as well. Something I wanted to come back and clean up. There is a precedent of not setting `Field(void *) varName = nullptr` -- was wondering if there's a reason for that. I'd like to set this to `nullptr`.